include/compartment.h

```diff
@@ -29,11 +29,19 @@
 // about
 
 struct func_intercept;
+void compartment_transition_out();
+int64_t comp_exec_in(void*, void* __capability, void*);
+void comp_exec_out();
+
+#define INTERCEPT_INSTR_COUNT 5
```

Comment:
Doc comments on these `#define`s would be good too.

Reply:
Added in https://github.com/capablevms/CHERI-ELF-comp/pull/2/commits/372ff860371648c042d36895047aecae22bee30e.